tests/suites/test_suite_x509parse.data

```diff
@@ -1658,10 +1658,6 @@ X509 CRT ASN1 (TBS, valid IssuerID, inv SubjectID, inv tag)
 depends_on:!MBEDTLS_X509_ALLOW_EXTENSIONS_NON_V3:MBEDTLS_RSA_C:MBEDTLS_SHA256_C
 x509parse_crt:"30819a308184a0030201008204deadbeef300d06092a864886f70d01010b0500300c310a30080600130454657374301c170c303930313031303030303030170c303931323331323335393539300c310a30080600130454657374302a300d06092A864886F70D010101050003190030160210ffffffffffffffffffffffffffffffff0202ffffa1000500300d06092a864886f70d01010b0500030200ff":"":MBEDTLS_ERR_X509_INVALID_FORMAT + MBEDTLS_ERR_ASN1_LENGTH_MISMATCH
 
-X509 CRT ASN1 (TBSCertificate v3, ext SubjectAlternativeName malformed)
```

Comment:
Why remove this testcase?

Reply:
Test coverage was extended.
`SubjectAlternativeName malformed` isn't too specific - in fact it had an empty `OtherName` component.
A lot of more specific cases have been added, and t he same test is performed by `X509 CRT ASN1 (TBS, inv SubjectAltName, otherName component empty)`.